usr/share/rear/restore/NBKDC/default/40_restore_backup.sh

```diff
@@ -0,0 +1,16 @@
+LogPrint "
+
+The System is now ready for restore. Please start the restore task from the NovaBACKUP DataCenter Central Management!
```

Comment:
There is a way and quite frankly, I was thinking of implementing it that way initially.
It is just a little harder, but still doable, as the actual "DataMover" or backup engine is scriptable.
The biggest factor is to keep the backup metadata or backup info record, along with the recovery image. Unfortunately the recovery iso image is written before the needed metadata is. 
Unfortunately there is no way to externally dig or select the backup metadata from the central server through an external interface. So we cannot query the central server who keeps the backup records in its database too.

So you see, it is possible, just a little more complicated. So that is why in this first implementation we will stick to this smpler way and perhaps at a later stage, convert to a full automatic recovery process.